test/quantization/eager/test_quantize_eager_ptq.py

```diff
@@ -943,6 +943,19 @@ def test_convtranspose_per_channel_qconfig_none(self):
         m[0].qconfig = None
         mp = torch.ao.quantization.prepare(m)
 
+    @skipIfNoFBGEMM
+    def test_quantwrapper_attaches_qconfig_to_dequant(self):
```

Comment:
just to confirm the logic for DeQuantStub, it is swapped to nnq.DeQuantize() only when qconfig is not None right?

Reply:
yes, that is correct